---
title: Nature Immunology - Single-cell landscape of immunological responses in patients with COVID-19
tags: TeXt
show_date: true
pageview: true
sharing: true
show_author_profile: true
---

__*Nature Immunology*__ : __Single-cell landscape of immunological responses in patients with COVID-19__

At the beginning of 2020, the coronavirus disease 2019 (__COVID-19__) epidemic broke out rapidly around the world. According to the latest statistics of the World Health Organization (__WHO__), as of August 11, 2020, the COVID-19 has caused a total of nearly 20 million confirmed cases and more than 730,000 deaths worldwide [1]. The COVID-19 caused by severe acute respiratory syndrome coronavirus 2 (__SARS-CoV-2__) infection results in a spectrum of illness from mild to severe disease and death. The viral infection and the antiviral host immune response interact and shape disease severity as well as clinical outcome. Therefore, the immunopathology of COVID-19 has received much attention. However, a global characterization of the protective or pathogenic immune responses in COVID-19 patients with different clinical settings is still lacking.
On August 12th, 2020, Nature Immunology published “Single-cell landscape of immunological responses in patients with COVID-19” from the laboratories of __Prof. Fan Bai__ and __Prof. Fu-Sheng Wang__. 

![Fig.0](https://raw.githubusercontent.com/SeadonXing/SeadonXing.github.io/master/docs/assets/images/BlogPictures/NI/Fig.0.png)

In this study, the research team performed single-cell RNA sequencing in peripheral blood samples of 5 healthy donors and 13 COVID-19 patients, including 7 patients with moderate, 4 patients with severe disease and 6 convalescent patients of whom 4 were paired with moderate cases (__Figure 1__). Through determining the transcriptional profiles of immune cells, coupled with assembled T cell receptor and B cell receptor sequences, the researchers analyzed the functional properties of immune cells and depicts a high-resolution transcriptomic landscape of blood immune cells during disease progression of COVID-19.

![Fig.1](https://raw.githubusercontent.com/SeadonXing/SeadonXing.github.io/master/docs/assets/images/BlogPictures/NI/Fig.1.png)

__Figure 1: Study design and single-cell transcriptional profiling of PBMCs from HDs and patients with COVID-19.__

First, after the unified single-cell analysis pipeline, transcripts were obtained from 122,542 cells from PBMCs of all samples. Using graph-based clustering method, the transcriptomes of 14 major cell types or subtypes were identified. These cells included T cells, B cells, natural killer (NK) cells, Monocytes and Dendritic cells et al. (__Figure 1__). Broad immune activation was observed in moderate and severe patients, evidenced by increased proportions of activated T, Pro T and plasma B cells, and decreased proportions of naïve T and Mono DC compartments (__Figure 2__). Though most of the clinical parameters restored to normal range at their convalescence, the state of the T-cell response was not fully restored, exemplified by the ratios of naïve T and Treg subsets. Next, by investigating the antiviral and pathogenic immune responses during SARS-CoV-2 infection, the researchers found that COVID-19 patients showed a concerted and strong interferon alpha response, an overall acute inflammatory response, and an enhanced migration ability, which peaked in most cell types of PBMCs from severe patients (__Figure 2__).

![Fig.2](https://raw.githubusercontent.com/SeadonXing/SeadonXing.github.io/master/docs/assets/images/BlogPictures/NI/Fig.2.png)

__Figure 2: Differences in cell composition and status across disease conditions.__

To further characterize changes in individual T cell subsets, 55,655 T cells were subclustered into 12 subsets. The proportions of active state T clusters were significantly higher in COVID-19 patients compared to healthy donors, including CD4+ Effector-GNLY, CD8+ Effector-GNLY, NKT CD56 and NKT CD160. Notably, the proportions of naive-state T cell subsets, including CD4+ naive, CD4+ memory, CD4+ effector memory, Treg, CD8+ naive and NKT naive subsets, decreased in patients with COVID-19 in comparison with HDs. Even in the conv condition, the proportions of CD4+ naive, CD8+ naive and Treg clusters did not restore to the levels of HDs. Interestingly, there is an NKT CD160 cluster, a previously described  NKT or Vδ1 T cell subset [2-5], served as an associated factor for effective anti-viral immunity, defined by clinical outcomes, in moderate patients (__Figure 3__). For the states of T cells, higher cytotoxicity of effector T cells were found in moderate patients, whereas higher exhaustion levels were seen in severe patients (__Figure 3__).
By reconstructing TCR sequences, the researchers found that clonal expansion was obvious in patients with COVID-19. The extent of clonal expansion in the moderate and conv conditions was higher than that of the severe condition. Meanwhile, large clonal expansions (clonal size >100) were absent in the severe condition (__Figure 4__), indicating that severe patients might lack efficient clonal expansion of effector T cells.

![Fig.3](https://raw.githubusercontent.com/SeadonXing/SeadonXing.github.io/master/docs/assets/images/BlogPictures/NI/Fig.3.png)

__Figure 3: Immunological features of T cell subsets.__

![Fig.4](https://raw.githubusercontent.com/SeadonXing/SeadonXing.github.io/master/docs/assets/images/BlogPictures/NI/Fig.4.png)

__Figure 4: Clonal expansion in T cells.__

For 11,377 B cells, six subsets were obtained by re-clustering (__Figure 5__). Notably, the proportions of active state B subsets, including germinal center B, plasma B and dividing plasma B subsets, increased in patients with COVID-19 in comparison with those of HDs. In contrast, the proportion of memory B cells decreased in patients with COVID-19. By reconstructing BCR sequences, the researchers found that B cells from severe patients showed obvious clonal expansions (__Figure 6__) than the other three conditions, indicating that B cell activity and humoral immune responses were strongly activated in severe patients. This raises the concern that, pathogen-directed antibodies can promote disease pathology, resulting in antibody-dependent enhancement.

![Fig.5-1](https://raw.githubusercontent.com/SeadonXing/SeadonXing.github.io/master/docs/assets/images/BlogPictures/NI/Fig.5-1.png)
![Fig.5-2](https://raw.githubusercontent.com/SeadonXing/SeadonXing.github.io/master/docs/assets/images/BlogPictures/NI/Fig.5-2.png)

__Figure 5: Immunological features of B cell subsets.__

![Fig.6](https://raw.githubusercontent.com/SeadonXing/SeadonXing.github.io/master/docs/assets/images/BlogPictures/NI/Fig.6.png)

__Figure 6: Expanded B cells in severe patients__

In summary, this integrated, multi-cellular description lays the foundation for future characterization of the complex, dynamic immune responses to SARS-CoV-2 infection. The transcriptomic data, coupled with detailed TCR, BCR-based lineage information, can serve as a rich resource for a better understanding of peripheral lymphocytes in COVID-19 patients and pave the way for rationally designed therapies as well as development of SARS-CoV-2-specific vaccines.

### __Author Info__

__Dr. Ji-Yuan Zhang, Xiang-Ming Wang, Xudong Xing__ and __Dr. Zhe Xu__ are the __co-first authors__ of this paper. __Prof. Fu-Sheng Wang__ at __Fifth Medical Center of Chinese PLA General Hospital__ and __Prof. Fan Bai__ at __BIOPIC and School of Life Sciences of Peking University__ are the __co-correspondence authors__. This work was supported by the National Key Research and Development Program of China (grant nos. 2020YFC0841900 and 2020YFC0844000) to F.-S.W. from the Ministry of Science and Technology of China.

### __Cite this article__

Zhang, J., Wang, X., Xing, X. et al. __Single-cell landscape of immunological responses in patients with COVID-19.__ __*Nat Immunol*__ 21, 1107–1118 (2020). <https://doi.org/10.1038/s41590-020-0762-x>

__References__

1. https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports

2. Pizzolato, G. et al. Single-cell RNA sequencing unveils the shared and the distinct cytotoxic hallmarks of human TCRVδ1 and TCR    Vδ2 γδ T lymphocytes. Proc. Natl Acad. Sci. USA 116, 11906–11915 (2019).

3. Couzi, L. et al. Common features of γδ T cells and CD8+ αβ T cells responding to human cytomegalovirus infection in kidney tran    splant recipients. J. Infect. Dis. 200, 1415–1424 (2009).

4. Déchanet, J. et al. Implication of γδ T cells in the human immune response to cytomegalovirus. J. Clin. Invest. 103, 1437–1449     (1999).

5. Farnault, L. et al. Clinical evidence implicating γδ T cells in EBV control following cord blood transplantation. Bone Marrow Tr   anspl. 48, 1478–1479 (2013)

<!--more-->

---

If you like, don't forget to [Follow and Star Me](https://github.com/SeadonXing?tab=stars). :star2:
